CMGTools/TTHAnalysis/cfg/run_susyMultilepton_cfg.py

```diff
@@ -331,9 +360,9 @@
         c.files = c.files[:nfiles]
         c.splitFactor = len(c.files)
         c.fineSplitFactor = 1
-        #c.splitFactor = 1; c.fineSplitFactor = 4
 
-if False: # For running on data
+
+if runData: # For running on data
     isData = True
```

Comment:
Why do we need to have both `isData` and `runData` flags ? can we have just one bool?